### ISSUE TITLE
e2e testing: searching Repos

### ISSUE THREAD

**Issue by bdougie** [open-sauced/hot#414]
Accessibility should be tested. Also, some simple functionality.

<img width="1044" alt="Screen Shot 2022-11-29 at 9 36 11 PM" src="https://user-images.githubusercontent.com/5713670/204716154-69207da3-a154-4d2c-b23d-da1a6a8a67bb.png">

**Comment by NsdHSO** [comment]
.take

**Comment by NsdHSO** [comment]
When I tested I received this screen 
![image](https://user-images.githubusercontent.com/37635083/205158484-c5c65804-7467-4404-975f-f6edff72ee39.png)
